events/tests/TESTS/events/equeue/CMakeLists.txt

```diff
@@ -1,18 +1,11 @@
-# Copyright (c) 2020 ARM Limited. All rights reserved.
+# Copyright (c) 2021 ARM Limited. All rights reserved.
```

Comment:
Thanks Lingkai, I actually meant to ask this - is it preferred to change the copyright date (and extend as you advised) or to leave it as is?

Reply:
Even 2020 is not accurate, because events has been there for many years since early days of Mbed. We added a copyright retrospectively last year, so the start date is okay I guess..